src/includes/classes/commands/class-versioncommand.php

```diff
@@ -123,6 +123,7 @@ public function get( $args, $assoc_args ) {
 				restore_current_blog();
 
 				if ( is_wp_error( $version ) ) {
+					/** @var WP_Error $version */
```

Comment:
Here and below: this reduces noise from Intelephense when it cannot infer the correct variable type.